packages/flutter/lib/src/widgets/virtual_viewport.dart

```diff
@@ -11,7 +11,9 @@ import 'package:flutter/rendering.dart';
 
 typedef void ExtentsChangedCallback(double contentExtent, double containerExtent);
 
+/// An abstract widget whose children are not all materialized.
 abstract class VirtualViewport extends RenderObjectWidget {
+  /// The offset at which the viewport should start painting children.
```

Comment:
offset from where? This should probably at least reference the anchor property and its type (which has the details).